_data/harnesses/confluent-parallel-consumer-application/kafka.yml

```diff
@@ -107,6 +107,12 @@ dev:
           render:
             file: tutorials/confluent-parallel-consumer-application/kafka/markup/dev/run-producer.adoc
 
+        - name: wait for producer
```

Comment:
mitigation for race condition i ran into (not sure if the change to `docker compose` caused this to fail more often. There were a few other build flakes that I fixed by adding or adjusting sleeps.